coderedcms/templatetags/coderedcms_tags.py

```diff
@@ -151,3 +133,21 @@ def richtext_amp(value):
 
     value = utils.convert_to_amp(value)
     return mark_safe(value)
+
+@register.filter
+def amp_formatting(value):
```

Comment:
Could this function be combined with `richtext_amp_formatting`? They seem to do the same thing, and it already checks for RichText instance?

Reply:
I could, but it wouldn't be really clean.  We're not quite checking whether the value is a `RichText` object.  When we need to ampify a rich text block, we need the rendered html from the `RichText` object.  We're not checking if the value in `richtext_amp_formatting` is or is not richtext, it assumes that.  

What we're checking for is whether the value is the internal richtext source (a pseudo template-like state) or a `RichText` object that typically wraps around the source.  We need the source to pass it through the `richtext` function which renders the source.  Then we pass through the fully rendered richtext html to `convert_to_amp`

The better solution would be to make `richtext_amp_formatting` a wrapper around `amp_formatting`, which I'll go ahead and do.